packages/calypso-products/src/products-list.ts

```diff
@@ -56,7 +56,10 @@ export type Product = {
 	getFeatures?: ( variation: string ) => string[];
 };
 
-export const JETPACK_PRODUCTS_LIST_WITH_FEATURES: Record< JetpackProductSlug, Product > = {
+export const JETPACK_SITE_PRODUCTS_WITH_FEATURES: Record<
```

Comment:
Renamed for clarity. WPCom search is Jetpack Search for wordpress.com, so both a Jetpack and WPCom product. This array contains products that are only available to Jetpack sites (therefore not WPCom search).